Mail/Views/Search/SearchFilterFolderCell.swift

```diff
@@ -25,12 +25,18 @@ import RealmSwift
 import SwiftUI
 
 struct SearchFilterFolderCell: View {
-    var folders: [Folder]
+    @StateObject private var viewModel: FolderListViewModel
```

Comment:
Do we need the ViewModel here? Maybe it should stay scope to the SearchableFolderListView